src/RTCRtpSender.ts

```diff
@@ -22,7 +24,7 @@ export default class RTCRtpSender {
         this._peerConnectionId = info.peerConnectionId;
         this._id = info.id;
         this._rtpParameters = new RTCRtpSendParameters(info.rtpParameters);
-
+        this._dtmf = new RTCDTMFSender(info.peerConnectionId);
```

Comment:
We should pass the sender ID here and then use it in the native side to match it.

Reply:
Also, what do browsers have as this property, for a video track, null?

Reply:
Done

Reply:
> Also, what do browsers have as this property, for a video track, null?

Not sure about this